intelmq/bots/parsers/eset/parser.py

```diff
@@ -0,0 +1,81 @@
+import json
+import re
+from intelmq.lib.utils import base64_encode, base64_decode
+from intelmq.lib.bot import ParserBot
+
+message_taxonomy_map = {
+    'Host actively distributes high-severity threat in the form of executable code.': 'malware',
+    'Host is known to be distributing low-risk and potentially unwanted content.': 'malware',
+    'Host is known source of phishing or other fraudulent content.': 'phishing',
+    'Host is known to be actively distributing adware or other medium-risk software.': 'other',
+    'Host is known source of active fraudulent content.': 'other'
+}
+
+
+class ESETParserBot(ParserBot):
+    def init(self):
+        self.f_map = {
+            'ei.urls (json)': self.urls_parse,
+            'ei.domains v2 (json)': self.domains_parse
+        }
+        self.pattern = re.compile('\\.'.join(['(25[0-5]|2[0-4][0-9]|1?[0-9]?[0-9])'] * 4))  # IPv4
+
+    def parse(self, report):  # yield single sections for parse_line to parse
```

Comment:
Looks the same as `parse_json`. You could instead just say `parse = ParserBot.parse_json`